src/planner/binder/tableref/bind_table_function.cpp

```diff
@@ -20,6 +20,8 @@
 #include "duckdb/catalog/catalog_entry/table_function_catalog_entry.hpp"
 #include "duckdb/catalog/catalog_entry/table_catalog_entry.hpp"
 #include "duckdb/function/table/read_csv.hpp"
+#include "duckdb/planner/expression/list.hpp"
```

Comment:
Can we remove the list includes and include only what we use?

Reply:
This seems to be a mistake from the last merge, I have not included these files myself. They are used below